src/apps/active-preview/components/Frame/Frame.less

```diff
@@ -14,6 +14,52 @@
   height: "100%";
 }
 
+.portrait {
+  transform: rotate(270deg);
+
+  .iframeContainer {
+    transform: rotate(90deg);
+
+    &.iphone14 {
+      width: 842px;
+      height: 389px;
+      margin-left: -227px;
```

Comment:
@agalin920 This is related to what Stuart also mentioned regarding rotating the parent container, so since the iframe gets rotated as well I had to rotate the iframe back 90deg again to make sure it's being viewed upright. But for some reason, the iframe gets pushed off frame, so had to add these weird margins to push it back into the phone's frame view again, tried doing it via flexbox but it doesn't really do anything.

Reply:
Has this been checked across different browsers? to ensure it behaves consistently

Reply:
@shrunyan yup, tested it on firefox and chromium-based browsers